install.sh

```diff
@@ -695,6 +696,7 @@ if [ $targetSystem = raspi ]
 then
     echo "Updating the Udev Rules"
     sudo bash $INSTALLERDIR/raspi/media_daemon/media_udev_rule.sh
+    sudo sed -i 's/slave/shared/' /lib/systemd/system/systemd-udevd.service
```

Comment:
What does this do? And why is it necessary? It is somehow surprising

Reply:
systemd-udevd creates its own filesystem namespace, so mount is done, but it is not visible in the principal namespace. Thus changing `MountFlags=slave` to `MountFlags=shared`.

Reply:
Updated the PR, Thanks for the suggestions

Reply:
Thanks for the info, I now read up on it, indeed. 

**BUT** Please don't change system files, please use the later approach mentioned here https://unix.stackexchange.com/questions/330094/udev-rule-to-mount-disk-does-not-work/330156#330156 the one where an override.conf is created. Thanks!

Reply:
I've done the mentioned changes, please review.